lib/backend/k8s/syncer.go

```diff
@@ -893,7 +899,7 @@ func (syn *kubeSyncer) parseNamespaceEvent(e watch.Event) []model.KVPair {
 	return []model.KVPair{*rules, *tags, *labels}
 }
 
-func (syn *kubeSyncer) parseNodeEvent(e watch.Event) *model.KVPair {
+func (syn *kubeSyncer) parseNodeEvent(e watch.Event) (kvpHostIp, kvpIPIPAddr *model.KVPair) {
```

Comment:
Can we remove the named returns? It's not something that shows up in the rest of the code and we've generally tried to avoid it.